src/plugin/plugin.rs

```diff
@@ -241,3 +269,221 @@ where
         }
     }
 }
+
+/// Specifies a default configuration for the default [`RapierContext`]
+///
+/// Designed to be passed as parameter to [`RapierPhysicsPlugin::with_default_world`].
+#[derive(Resource, Debug, Reflect, Clone)]
+pub enum RapierContextInitialization {
+    /// [`RapierPhysicsPlugin`] will not spawn any entity containing [`RapierContext`] automatically.
+    ///
+    /// You are responsible for creating a [`RapierContext`],
+    /// before spawning any rapier entities (rigidbodies, colliders, joints).
+    ///
+    /// You might be interested in adding [`DefaultRapierContext`] to the created world.
+    NoAutomaticRapierContext,
+    /// [`RapierPhysicsPlugin`] will spawn an entity containing a [`RapierContext`]
+    /// automatically during [`PreStartup`], with the [`DefaultRapierContext`] marker component.
+    InitializeDefaultRapierContext {
+        /// See [`IntegrationParameters::length_unit`]
+        length_unit: f32,
+    },
+}
+
+impl Default for RapierContextInitialization {
+    fn default() -> Self {
+        RapierContextInitialization::InitializeDefaultRapierContext { length_unit: 1f32 }
+    }
+}
+
+pub fn insert_default_world(
+    mut commands: Commands,
+    initialization_data: Res<RapierContextInitialization>,
+) {
+    match initialization_data.as_ref() {
+        RapierContextInitialization::NoAutomaticRapierContext => {}
+        RapierContextInitialization::InitializeDefaultRapierContext { length_unit } => {
+            commands.spawn((
```

Comment:
Giving this entity a Name component would be helpful, to let people easily see that it's the rapier context.